velox/exec/Window.cpp

```diff
@@ -52,8 +52,6 @@ Window::Window(
           operatorId,
           windowNode->id(),
           "Window"),
-      outputBatchSizeInBytes_(
-          driverCtx->queryConfig().preferredOutputBatchSize()),
```

Comment:
@mbasmanova preferredOutputBatchSize() used to mean # of rows, which is 1024. Seems like a bug here. So after my change, behavior will change. But I guess Window op is added recently and not being used a lot.

Reply:
@gggrace14 Ge, nice catch. 

CC: @aditi-pandit